### PR TITLE
Remove isShipped attribute

### PR DESCRIPTION
**Pull request by nickvergessen**
@DeepDiver1975 

owncloud/core#13706

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -5,7 +5,6 @@
 	<licence>AGPL</licence>
 	<author>Georg Ehrke, Bart Visscher, Jakob Sack</author>
 	<require>8</require>
-	<shipped>true</shipped>
 	<description>Calendar with CalDAV support</description>
 	<default_enable/>
```
Comment (georgehrke) [inline appinfo/info.xml]: please remove this line as well, this will cause issues when installing the app over the app store

Comment (nickvergessen) [inline appinfo/info.xml]: I assume this applies to all apps?

Comment (georgehrke) [inline appinfo/info.xml]: Ok, so there is no check inside [checkAppsIntegrity](https://github.com/owncloud/core/blob/9771ead7dd0f761e5379739d9bf53048ba602fe8/lib/private/installer.php#L286), but it's better to remove it anyway

Comment (nickvergessen) [inline appinfo/info.xml]: right, removed it

Comment (georgehrke) [inline appinfo/info.xml]: Dev doc says that it's for core apps only http://doc.owncloud.org/server/6.0/developer_manual/app/appframework/info.html#default-enable

**Comment by georgehrke** [comment]
:+1:

**Pull request merged**